bin.js

```diff
@@ -37,7 +37,8 @@ cli
 cli.command('authorize <email>')
   .alias('auth')
   .example('authorize user@example.com')
-  .describe('Authorize this agent to interact with the w3up service with any capabilities granted to the given email.')
+  .describe('Authorize this agent to interact with the w3up service with capabilities granted to the given email.')
+  .option('-c, --can', 'One or more abilities to authorize.', '*')
```

Comment:
I would prefer not to add a default of `*`. Can we add the flag separately from adding the default?
Not a hard blocker though. I asked for advice on [tradeoffs](https://github.com/orgs/ucan-wg/discussions/153)

Reply:
Ah yes, will change this now...